packages/api-plugin-carts/src/mutations/transformAndValidateCart.js

```diff
@@ -15,7 +15,7 @@ const logCtx = { name: "cart", file: "transformAndValidateCart" };
  */
 export default async function transformAndValidateCart(context, cart) {
   const { simpleSchemas: { Cart: cartSchema } } = context;
-  updateCartFulfillmentGroups(context, cart);
+  await updateCartFulfillmentGroups(context, cart);
```

Comment:
A little concerned about this. @aldeed was there a reason we didn't await these transforms?

Reply:
Earlier we did not have any async calls in the `updateCartFulfillmentGroups`. Now we have added a new async call to check if the fulfillment-type selected is currently enabled/disabled. That is why I added a new await here now.